Makefile

```diff
@@ -77,6 +77,15 @@ configgen:
 	echo "" >> conf/st2.conf.sample
 	. $(VIRTUALENV_DIR)/bin/activate; python ./tools/config_gen.py >> conf/st2.conf.sample;
 
+.PHONY: .generate-openapi-spec
```

Comment:
This tool st2-validate-api-spec has a --generate option. Under the hood, it does use generate_spect but it's better to use the binary IMO. Also, it was intentional I didn't add this to linter automatically. I didn't have a good place to include the generation. IMO, the lint target should just validate. So I'd do the following changes:

.lint: .lint-api-spec .flake8 .pylint

.lint-api-spec: .generate-api-spec .lint-openapi-spec

Am I making sense? This is slightly better than what you've right now. Only slightly.